spec/controllers/idv/capture_doc_status_controller_spec.rb

```diff
@@ -103,6 +103,16 @@
 
         expect(response.status).to eq(200)
       end
+
+      context 'when redo document capture' do
+        before do
+          idv_session[:redo_document_capture] = true
+        end
+        it 'resets redo_document capture in idv_session' do
+          get :show
+          expect(controller.user_session[:idv][:redo_document_capture]).to be_nil
```

Comment:
Wait, I made this pass by using `controller.idv_session` (that's what I meant!) and then making idv_session public rather than private in the controller. Ticket still stands, idv_session is not a hash.